red_queen/games/applications/MQT/bench/utils/utils.py

```diff
@@ -0,0 +1,158 @@
+from qiskit import QuantumCircuit, __qiskit_version__
+from qiskit.compiler import transpile
+from qiskit.transpiler import CouplingMap
+from qiskit.circuit import qpy_serialization
+from qiskit.algorithms import EstimationProblem
+
+from datetime import date
+
+import networkx as nx
+import numpy as np
+from os import path
+
+from qiskit.test.mock import (
+    FakeBogota,
+    FakeCasablanca,
+    FakeGuadalupe,
+    FakeMontreal,
+    FakeManhattan,
+    FakeWashington,
+)
+qasm_path = "/home/archana/qamp-fall-22/red-queen/red_queen/applications/games/qasm"
+def set_qasm_output_path(new_path: str = "/home/archana/qamp-fall-22/red-queen/red_queen/applications/games/qasm"):
```

Comment:
We should be using relative paths instead of an absolute path with your local home directory. Otherwise when I run this I'll get a filesystem error because this path doesn't exist.